testdata/py3_image.py

```diff
@@ -19,6 +21,9 @@ def main():
   print('Second: %d' % py3_image_library.fn(2))
   print('Third: %d' % py3_image_library.fn(3))
   print('Fourth: %d' % py3_image_library.fn(4))
+  print(sys.argv)
```

Comment:
can you add short doc line somewhere indicating these two tests (py3, py) expect the third argument to be a file location